### ISSUE TITLE
Does -watch actually work?

### ISSUE THREAD

**Issue by rhukster** [dailymuse/gosass#8]
The regular compile is fine, but i am running with this:

```
gosass -watch -input scss/ -output css-compiled/
```

And I cannot get it to recompile when i make a change to any of the `scss` files. Running the latest `gosass` as I just installed a few minutes ago.

**Comment by ysimonson** [comment]
You need to use a filesystem that supports inotify. `-watch` won't work in, e.g. NFS mounts and Vagrant shared folders. Is this the case with you?

**Comment by rhukster** [comment]
Actually, i'm not sure what I did, but now it's working.  It is a regular mac filesystem, only non-standard thing is that case-sensitivity is enabled.  

It wasn't working, then I tried adding a `-load-path` and that worked even though my path was not valid, and imports failed.  Then I removed the the `-load-path` and it worked fine without it.  Really not sure what caused it start working, but I won't complain though!
